inst/rmarkdown/templates/problemset_pdf/resources/template.tex

```diff
@@ -0,0 +1,125 @@
+\documentclass[11pt,a4paper]{article}
+\usepackage[top=1.3cm,bottom=2cm,left=2cm,right=2cm]{geometry}
+
+\usepackage[immediate]{silence}
+\WarningFilter[temp]{latex}{Command} % silence the warning
+\usepackage{sectsty}
+\DeactivateWarningFilters[temp] % So nothing unrelated gets silenced
+\sectionfont{\fontsize{12}{15}\selectfont}
+\subsectionfont{\fontsize{10}{12}\selectfont}
+\allsectionsfont{\sffamily} % Set font of section
+
+\usepackage{calc}
+%%%%% FONTS
```

Comment:
Fonts are fine now, but there is an undefined control sequence.

```
! Undefined control sequence.
l.240 (Intercept) & \num
                        {7948.667} & \num{8.241} & \num{16259.384} & \num{9....

Error: LaTeX failed to compile MINT001_ProblemSet_2021-08-21.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See MINT001_ProblemSet_2021-08-21.log for more info.
```

Either you need to define `\num` somewhere in the preamble, or seek out another, more LaTeX-friendly printing solution.

Reply:
Strange, this issue is OS-specific as it doesn't appear on my machine running Windows with `MikTex 21.6` installed. I have pushed some changes reducing the number of packages mentioned in the Latex template which sped up the rendering and will hopfeully solve this issue as well.
If the issue persists, could you send me the log file generated with the PDF so that I can see what this \num{} sequence is?

Reply:
Here: [MINT001_ProblemSet_2021-08-21.log](https://github.com/jhollway/iheiddown/files/7030187/MINT001_ProblemSet_2021-08-21.log)

Reply:
Thanks for the log file! I added the `{siunitx}` package to the preamble of the Latex command which will define the \num{} environment and display the numbers in the table correctly. Could you confirm that it works on macOS when you have the chance @jhollway ?